package.json

```diff
@@ -30,6 +30,7 @@
     "wellknown": "^0.5.0"
   },
   "devDependencies": {
+    "@types/react-test-renderer": "^18.0.0",
```

Comment:
Curious how are we using this dependency?

Reply:
You mean where is this used in my test? 
https://github.com/opensearch-project/dashboards-maps/pull/242/files#diff-a541d7c1c45c0863a23f2597da09b9770059d8686cff659d192b4447bdaffee1R9